mlflow/pytorch/_pytorch_autolog.py

```diff
@@ -108,12 +108,25 @@ def _log_metrics(self, trainer, pl_module):
                     if isinstance(callback, pl.callbacks.early_stopping.EarlyStopping):
                         self._early_stop_check(callback)
 
+            _pl_version = Version(pl.__version__)
```

Comment:
Assign `Version(pl.__version__)` into a variable to avoid parsing multiple times (not a big issue though)